File: bddtests/src/main/kotlin/mocking/vision/Demographics/PrimaryAddress.kt
package mocking.vision.Demographics

import javax.xml.bind.annotation.XmlElement
import javax.xml.bind.annotation.XmlRootElement

@XmlRootElement(name="primaryAddress", namespace = "urn:vision")
data class PrimaryAddress(
    @XmlElement(namespace="", name="houseName")
    var houseName: String? = null,
    @XmlElement(namespace="", name="houseNumber")
    var houseNumber: String? = null,
    @XmlElement(namespace="", name="street")
    var street: String? = null,
    @XmlElement(namespace="", name="town")
    var town: String? = null,
    @XmlElement(namespace="", name="county")
    var county: String? = null,
    @XmlElement(namespace="", name="postcode")
    var postcode: String? = null
)
{
    constructor() : this("", "", "", "")
}

File: backendworker/NHSOnline.Backend.GpSystems/Suppliers/Vision/Models/PatientRecord/PrimaryAddress.cs
using System.Xml.Serialization;

namespace NHSOnline.Backend.GpSystems.Suppliers.Vision.Models.PatientRecord
{
    public class PrimaryAddress 
    {    
        [XmlElement(ElementName="houseName", Namespace="urn:vision")]
        public string HouseName { get; set; }
        
        [XmlElement(ElementName="houseNumber", Namespace="urn:vision")]
        public string HouseNumber { get; set; }
        
        [XmlElement(ElementName="street", Namespace="urn:vision")]
        public string Street { get; set; }
        
        [XmlElement(ElementName="town", Namespace="urn:vision")]
        public string Town { get; set; }
        
        [XmlElement(ElementName="county", Namespace="urn:vision")]
        public string County { get; set; }
        
        [XmlElement(ElementName="postcode", Namespace="urn:vision")]
        public string Postcode { get; set; }

    }
}